foolbox/models/base.py

```diff
@@ -66,7 +66,10 @@ def _preprocess(self, inputs: ep.TensorType) -> ep.TensorType:
 
     @property
     def data_format(self) -> Any:
-        return getattr(self._model, "data_format", None)
+        if hasattr(self._model, "data_format"):
```

Comment:
Sorry, my last advice was wrong. Mypy doesn't actually support this (inference using `isinstance` checks work, but `hasattr` checks don't).
But I just realized that these changes don't make sense anyway. The whole function should simply be `self._model.data_format` (plus a type ignore comment) or the original `getattr` without the `None` default value. The `AttributeError` will automatically be propagated, so why manually raising it?

Reply:
@zimmerrol Reminder: always run Mypy locally

Reply:
Oh yes, you're right.